tests/Unit/Elliptic/DiscontinuousGalerkin/Test_DgOperator.cpp

```diff
@@ -383,256 +387,341 @@ SPECTRE_TEST_CASE("Unit.Elliptic.DG.Operator", "[Unit][Elliptic]") {
+          0.34345601404832493, 0.5131281541990022,   0.6666245501640716,
+          0.10590848505681383, 0.13089495066408074,  0.32198060646830806,
+          0.6615643366662437,  0.8465062252707221,   0.5532573447967134,
+          0.8544524875245048,  0.3848378112757611,   0.31678789711837974,
+          0.3542646755916785,  0.17108182920509907,  0.8291126345018904,
+          0.3386708459143266,  0.5523700752940731,   0.578551468108833,
+          0.5215330593973323,  0.002688064574320692, 0.98834541928282};
+      Vars vars_rnd_neighbor_eta{24};
+      get(get<Var<Poisson::Tags::Field>>(vars_rnd_neighbor_eta)) = DataVector{
+          0.5194851192598093, 0.6128945257629677,  0.12062866599032374,
+          0.8263408005068332, 0.6030601284109274,  0.5450680064664649,
+          0.3427638337743084, 0.3041207890271841,  0.4170222110247016,
+          0.6813007657927966, 0.8754568417951749,  0.5104223374780111,
+          0.6693137829622723, 0.5859365525622129,  0.6249035020955999,
+          0.6746890509878248, 0.8423424376202573,  0.08319498833243877,
+          0.7636828414433382, 0.243666374536874,   0.19422296057877086,
+          0.5724569574914731, 0.09571251661238711, 0.8853268262751396};
+      Vars vars_rnd_neighbor_zeta{24};
+      get(get<Var<Poisson::Tags::Field>>(vars_rnd_neighbor_zeta)) = DataVector{
+          0.7224433825702216,  0.3229589138531782,  0.3617886556223141,
+          0.22826323087895561, 0.29371404638882936, 0.6309761238544878,
+          0.09210493994507518, 0.43370117267952824, 0.4308627633296438,
+          0.4936850976503062,  0.425830290295828,   0.3122612229724653,
+          0.4263513069628082,  0.8933891631171348,  0.9441600182038796,
+          0.5018366758843366,  0.6239529517921112,  0.11561839507929572,
+          0.3172854818203209,  0.4148262119536318,  0.8663091578833659,
+          0.2504553653965067,  0.48303426426270435, 0.985559785610705};
+      OperatorVars expected_operator_vars_rnd{24};
+      get(get<DgOperatorAppliedTo<Var<Poisson::Tags::Field>>>(
+          expected_operator_vars_rnd)) = DataVector{
+          618.6142450194411,  269.8213716601356,   49.33225265133292,
+          103.71967654882658, 219.4353476547795,   -14.237023651828594,
+          731.9842766450536,  490.9303825979318,   32.18932195031287,
+          13.87090223491767,  -13.954381736466516, 130.61721549991918,
+          331.75024822120696, 55.511704965231125,  17.52350289937635,
+          23.878697549520762, -183.34493489042083, -171.66677910143915,
+          390.19201603025016, 410.25585855100763,  53.690124372228034,
+          82.23683297149915,  53.091014251828675,  117.36921898587735};
+      Approx analytic_solution_aux_approx =
+          Approx::custom().epsilon(8.e-1).scale(M_PI);
+      Approx analytic_solution_operator_approx =
+          Approx::custom().epsilon(8.e-1).scale(M_PI * penalty_parameter *
+                                                square(4) / 2.);
```

Comment:
I'm still worried that you need to accept 80% errors in this case (and slightly less above).  Why is this result so inaccurate?

Reply:
Well this 3D case tries to resolve a full sinusoid period with 2 linear elements :) At x=-0.5 the gradient is zero analytically, but 2 numerically because the sinusoid is approximated as a ^ shape (or upside-down V, i.e. just two lines with slope 2 and -2). That's a large error. I chose so few elements to avoid writing even more numbers in the regression test. The added code below repeats the analytic-solution test with much higher resolution. Note that the low tolerance here only applies to the analytic-solution test; the custom values are checked to default precision (1.-14 or so).

Reply:
Can you set the expected values to what you actually expect the operator to return, then?

Reply:
Yea that's what the regression test does on the low-resolution domain. I could disable the analytic-solution test on the low-resolution domain by making the `approx` arguments `std::optional`s, so the comparison to the analytic solution only runs on the high-resolution domain. Not sure if that's worth it though, it just deletes these low-resolution/high-tolerance analytic-solution tests.

Reply:
Sorry, you've lost me.  This *is* the regression test, isn't it?

Reply:
Yes it is. It compares the hard-coded numbers to operator output at default precision (1e-14 or so). It is at low resolution to keep the amount of hard-coded numbers low, and because the resolution is irrelevant for the regression test. It also does an analytic-solution test because that's built-in to the testing function, but does it at very low resolution and thus very high tolerance. Therefore, as response to your comment, I added another test at much higher resolution below. That one doesn't test any hard-coded numbers but only the analytic solution at much smaller tolerance.

Reply:
Oh, I see now.  Could you add a comment explaining that these loose approx values are effectively disabling a part of the test you don't want, but there is another part running with more typical tolerances?  I didn't understand that and thought these were the only tolerances being used.

Reply:
Done! Pushed another fixup.

Reply:
I think that's clear, now.  Go ahead and squash.